prime-router/src/main/kotlin/Report.kt

```diff
@@ -25,6 +25,33 @@ typealias DefaultValues = Map<String, String>
  * unique id and name as well as list of sources for the creation of the report.
  */
 class Report {
+    enum class Format {
+        INTERNAL, // A format that serializes all elements of a report (A CSV today)
+        CSV, // A CSV format the follows the csvFields
+        HL7, // HL7 with one result per file
+        HL7_BATCH, // HL7 with BHS and FHS headers
+        REDOX; // Redox format
+        // FHIR
+
+        fun toExt(): String {
+            return when (this) {
+                INTERNAL -> "internal"
+                CSV -> "csv"
+                HL7 -> "hl7"
+                HL7_BATCH -> "hl7"
+                REDOX -> "redox"
+            }
+        }
+
+        fun isSingleItemFormat(): Boolean {
+            return when (this) {
+                REDOX -> true
+                HL7 -> true
+                else -> false
+            }
+        }
+    }
+
     /**
```

Comment:
Moving Format into Report from OrganizationService. This is a step in the right direction. I'd like to explore how it could be a property of Schema, but that is something for later.

Reply:
I don't know. I kind of like the idea that a schema can be chimeric and handle different output formats. I see merits to doing it both ways, so I'm not opposed to having the format in the schema if everyone wants to go that way, but if do go that direction, then we may need to revisit how we're doing the `csvFields` logic. It could need to be refactored if a schema is only ever used for CSV serialization.